src/StochDynamicProgramming.jl

```diff
@@ -1,27 +1,35 @@
-#  Copyright 2015, Vincent Leclere, Francois Pacaud and Henri Gerard
+#  Copyright 2017, V.Leclere, H.Gerard, F.Pacaud, T.Rigaut
 #  This Source Code Form is subject to the terms of the Mozilla Public
 #  License, v. 2.0. If a copy of the MPL was not distributed with this
 #  file, You can obtain one at http://mozilla.org/MPL/2.0/.
 #############################################################################
 # SDDP is an implementation of the Stochastic Dual Dynamic Programming
 # algorithm for multi-stage stochastic convex optimization problem
-# see TODO
 #############################################################################
+
+__precompile__()
+
 module StochDynamicProgramming
 include("SDPutils.jl")
 
-using MathProgBase, JuMP, Distributions
+using MathProgBase, JuMP, Distributions, StochasticDualDynamicProgramming
+using DocStringExtensions
+using CutPruners
```

Comment:
If CutPruners is independant we should try to have a mechanism to be able to use the package without it